### PR TITLE
TSLint: enable rule 'react/no-unstable-nested-components'

### PR DESCRIPTION
**Pull request by alfonsomthd**
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md

### CODE REVIEW

File: packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx [hunk 1]
```diff
@@ -178,16 +208,13 @@ export const NamespaceSelectionTable: React.FC<NamespaceSelectionTableProps> =
...
-                    : t('There are no namespaces to display.'))}
-              </Bullseye>
-            )}
+            emptyRowMessage={
+              searchError
+                ? null
+                : clusterName
+                ? EmptyRowMessage
+                : NoClusterEmptyRowMessage
+            }
```
Comment (SanjalKatiyar) [inline packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx:217]:
```suggestion
            emptyRowMessage={
              searchError
                ? clusterName
                ? EmptyRowMessage
                : NoClusterEmptyRowMessage
                : null
            }
```

Comment (SanjalKatiyar) [reply packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx:217]: plz ignore the comment...

**Comment by SanjalKatiyar** [comment]
LGTM, cc @GowthamShanmugam for tagging.

File: packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx [hunk 2]
```diff
@@ -178,16 +208,13 @@ export const NamespaceSelectionTable: React.FC<NamespaceSelectionTableProps> =
...
-                      )
-                    : t('There are no namespaces to display.'))}
-              </Bullseye>
-            )}
+            emptyRowMessage={
+              searchError
+                ? null
+                : clusterName
+                ? EmptyRowMessage
+                : NoClusterEmptyRowMessage
```
Comment (GowthamShanmugam) [inline packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx:216]:
```suggestion
              searchError
                ? searchError
                : clusterName
                ? EmptyRowMessage
                : NoClusterEmptyRowMessage
```

if a search error is found, we need to display searchError.

File: packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx (hunk 1 above)
Comment (GowthamShanmugam) [reply packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx:217]: https://github.com/red-hat-storage/odf-console/pull/1663/files#r1837622245

File: packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx (hunk 2 above)
Comment (alfonsomthd) [reply packages/mco/components/discovered-application-wizard/wizard-steps/namespace-step/namespace-table.tsx]: Done.

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**